cli/internal/runsummary/task_summary.go

```diff
@@ -30,15 +30,18 @@ type TaskSummary struct {
 	ResolvedTaskDefinition *fs.TaskDefinition                    `json:"resolvedTaskDefinition"`
 	ExpandedOutputs        []turbopath.AnchoredSystemPath        `json:"expandedOutputs"`
 	Framework              string                                `json:"framework"`
+	EnvMode                util.EnvMode                          `json:"envMode"`
 	EnvVars                TaskEnvVarSummary                     `json:"environmentVariables"`
 	Execution              *TaskExecutionSummary                 `json:"execution,omitempty"` // omit when it's not set
 }
 
 // TaskEnvVarSummary contains the environment variables that impacted a task's hash
 type TaskEnvVarSummary struct {
-	Configured []string `json:"configured"`
-	Inferred   []string `json:"inferred"`
-	Global     []string `json:"global"`
+	Configured        []string `json:"configured"`
+	Inferred          []string `json:"inferred"`
+	Global            []string `json:"global"`
+	Passthrough       []string `json:"passthrough"`
+	GlobalPassthrough []string `json:"globalPassthrough"`
```

Comment:
Please bikeshed naming.

Reply:
How about 
- `allowList` and `globalAllowList`? 
- `allowed` and `globalAllowed` (which follows the pattern of the configired/inferred)

Reply:
We use "passthrough" in the config key, so I believe that continuing to do so is valuable.